lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
```

Comment:
I've just noticed that this entire section was duplicated from above. I'd rather see the code refactored so that it can be output twice (if that's what is really needed here). Is there a better UX than just showing two near identical forms?